src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/authentication/JwtUtil.java

```diff
@@ -24,7 +25,7 @@ public class JwtUtil implements Serializable {
   private static final long serialVersionUID = -2550185165626007488L;
   private final String secret;
 
-  public JwtUtil(@Value("${jwt.secret}") final String secret) {
+  public JwtUtil(@Value("${jwt.secret}") final String secret, UserDataService userDataService) {
```

Comment:
Why this change?

Reply:
Removed it, it was just an oversight from my testing.